testutils/inject/app_service_client.go

```diff
@@ -12,8 +12,16 @@ type AppServiceClient struct {
 	apppb.AppServiceClient
 	ListOrganizationsFunc func(ctx context.Context, in *apppb.ListOrganizationsRequest,
 		opts ...grpc.CallOption) (*apppb.ListOrganizationsResponse, error)
+	ListLocationsFunc func(ctx context.Context, in *apppb.ListLocationsRequest,
+		opts ...grpc.CallOption) (*apppb.ListLocationsResponse, error)
+	ListRobotsFunc func(ctx context.Context, in *apppb.ListRobotsRequest,
+		opts ...grpc.CallOption) (*apppb.ListRobotsResponse, error)
 	CreateKeyFunc func(ctx context.Context, in *apppb.CreateKeyRequest,
 		opts ...grpc.CallOption) (*apppb.CreateKeyResponse, error)
+	GetRobotPartsFunc func(ctx context.Context, in *apppb.GetRobotPartsRequest,
+		opts ...grpc.CallOption) (*apppb.GetRobotPartsResponse, error)
+	GetRobotPartLogsFunc func(ctx context.Context, in *apppb.GetRobotPartLogsRequest,
+		opts ...grpc.CallOption) (*apppb.GetRobotPartLogsResponse, error)
```

Comment:
Needed a lot more inject functionality to actually call `GetRobotPartLogs` without a panic from the CLI.